src/components/Button/Button.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import BUTTON from '../typography/BUTTON/BUTTON'
+
+const CustomButton = styled.button`
+  border: solid 2px ${props => props.theme.color('secondary', 'bright')};
+  color: ${props => props.theme.color('secondary', 'bright')};
+  border-radius: 1.3rem;
+  padding: 1rem;
+  background-color: transparent;
+
+  &:first-child {
```

Comment:
This CSS rule shouldn't belong to `Button` component. This must be moved to the component where the buttons are presented (in this case `CtaButtons`).